docs/CHANGELOG.rst

```diff
@@ -4,6 +4,14 @@ feflow Change Log
 
 .. current developments
 
+v0.1.1
+====================
+
+- Support for ``pydantic`` >=1.10.17, allowing ``pydantic`` 2 to coexist in the same environment (`PR #58 <https://github.com/OpenFreeEnergy/feflow/pull/58>`_).
```

Comment:
It might be good at some point to switch the changelog to using rever (that way we have the same development & deployment workflow everywhere in the org).

Reply:
We do have a `rever.xsh` basic setup here, but I haven't had the time to figure out how to make it fill the log, so I just did it myself. But definitely, that is something we probably want to properly setup. Thanks!